source/_components/ebusd.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: "ebusd"
+description: "The ebusd component allows the integration between eBUS heating system and Home Assistant."
+date: 2018-12-27 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Ebusd
```

Comment:
This is a non-existing category. Please check our current documentation for a list.

Reply:
This is a new one, I have created a new component inside component/ebusd folder. Which is the correct category otherwise?

Reply:
I would say: Sensor
Since it creates sensors for the ebusd heating systems.

Reply:
yes, but it also have a service so it is not only a sensor. If you confirm I will make this correction anyway

Reply:
As per home-assistant/architecture#131, sensors should not have services.

Reply:
That's why I have created my own component named ebusd. That's also a category, correct?

Reply:
No, it is not. Go to our website, hit the components page, categories are on your left menu. It would be weird to see a `ebusd` there.